packages/office-ui-fabric-react/src/components/Stack/Stack.types.ts

```diff
@@ -84,6 +84,7 @@ export interface IStackProps
    * Defines the spacing between Stack children.
    * The property is specified as a value for 'row gap', followed optionally by a value for 'column gap'.
    * If 'column gap' is omitted, it's set to the same value as 'row gap'.
+   * @deprecated Use `childrenGap` token instead.
```

Comment:
We should add a `warnDeprecations` call for this too. I'm not sure how difficult that is since we're not using `BaseComponent`, but maybe there's an underlying utility function we can call directly...

Reply:
Perhaps https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/utilities/src/warn/warnDeprecations.ts?